docs/tutorial/writing-a-more-complex-dataset-definition/index.md

```diff
@@ -383,39 +383,53 @@ Notice that we:
 
 ### Outcome variables
 
-#### Date of first admission
+#### Date of first asthma exacerbation
 
-First, we import the `apcs` table.
+
+We use the
+[Asthma Exacerbations](https://www.opencodelists.org/codelist/bristol/asthma-exacerbations/73c4eace/)
+codelist to query the `clinical_events` table.
+
+The codelist is stored in `codelists/bristol-asthma-exacerbations.csv`.
+If we open the CSV file,
+then we see that the `code` column contains the codes.
+
+First, we use the `codelist_from_csv` function to read the CSV file.
 
 ```python
-from ehrql.tables.tpp import apcs
+asthma_exacerbations_codelist = codelist_from_csv(
+    "codelists/bristol-asthma-exacerbations.csv",
+    column="code",
+)
 ```
 
 Finally, we query the table
```

Comment:
It reads a bit oddly to say "First..." on line 397, and then "Finally,..." as the next thing (I suspect this was copy/pasted originally from the Ethnicity or Number of medication sections which both have a First (codelist), Next, Finally pattern.